### PR TITLE
Feature/Design 106 browser notification

### PR DESCRIPTION
**Pull request by TrebBrennan**
This is an all purpose CSS style for these types of messages, it could also be used for No-JS, no cookies, etc, depending on the needs of teams implementing UI-Kit.
## Note

Currently this text is intended for internal demos and will need to change for Beta releases to better explain levels of support
## Definition of Done
- [x] Content/documentation reviewed by Julian or someone in the Content Team
- [x] UX reviewed by Gary or someone the Design team
- [x] Code reviewed by one of the core developers
- [x] Acceptance Testing
  - [x] Cross-browser tested ie7-9, Chrome
  - [x] HTML5 validation (CircleCI)
- [ ] Stakeholder/PO review
- [ ] CHANGELOG updated

### CODE REVIEW

**Comment by TrebBrennan** [comment]
screen shot 2016-08-19 at 12 15 59 pm

File: assets/sass/_accessibility.scss [hunk 1]
```diff
@@ -16,3 +16,16 @@
   position: absolute;
   width: 1px;
 }
+
+/*
+  Warning/notification regarding feature support
+*/
+.legacy-msg {
```
Comment (klepas) [inline assets/sass/_accessibility.scss:23]: ‘legacy’ might not be the most apt considering the browser support philosophy — what do you reckon @TrebBrennan ?

**Comment by klepas** [comment]
Made one comment re. the choice of class name; else LGTM. (:

**Comment by TrebBrennan** [comment]
@klepas what would you suggest?

**Comment by klepas** [comment]
@TrebBrennan maybe `.browser-support-msg`?

**Comment by TrebBrennan** [comment]
@klepas Was intending for it to be generic (eg for JS too):
`<noscript><p class="legacy-msg">Note here</p></noscript>`

What do you think about `.feature-support-msg` ? or `support-msg` ?

**Comment by klepas** [comment]
@TrebBrennan In that case I’d be tempted to make it `ua-support-msg` (for user agent)?

**Comment by gazhands** [comment]
Should we show Gov.au page underneath at all?

**Comment by TrebBrennan** [comment]
Closing PR for a sec to resolve above comment from @gazhands.

**Pull request closed without merging**